Bring in the pods!

vuiets opened the pull request:
Huzzah! **Pods** are here starting with two
- **Mine** - to mine stones and clear weeds in the mines
- **Battle** - to defend you from evil monsters in the mines

vuiets committed:
shaky addition of pods

vuiets committed:
remove unused DroneManager class

vuiets committed:
fix horrible issue

vuiets committed:
add pod configurations

vuiets committed:
add battle pod

vuiets committed:
fixed mine pod

vuiets committed:
fixed battle pod

vuiets committed:
smoothening pod actions

wizvishak commented:
**To do:**
- Fix and update the pod configs
- Fix README in #4 
- Add code comments
- Bump version

vuiets committed:
defer breaking mine containers

vuiets committed:
fix README.md issues

vuiets committed:
code comments for all framework files

missed out code comments for framework files

missing code comments for NPCOptions class

vuiets committed:
code comments in mod entry and config

vuiets reviewed (commented):
A lazy review.

wizvishak commented:
Fixes #4
